src/item_category.h

```diff
@@ -4,12 +4,24 @@
 
 #include <string>
 
+#include "flat_set.h"
 #include "optional.h"
 #include "translations.h"
 #include "type_id.h"
 
+class item;
 class JsonObject;
 
+// this is a helper struct with rules for picking a zone
+struct zone_priority_data {
+    bool was_loaded;
+    zone_type_id id;
+    bool filthy = false;
+    cata::flat_set<std::string> flags;
+
+    void deserialize( JsonIn &jsin );
+    void load( JsonObject &jo, const std::string & = "" );
```

Comment:
```suggestion
    void load( JsonObject &jo, const std::string & );
```

Reply:
turns out i can just remove the string parameter all together.